charts/metering-ansible-operator/templates/crds/reportquery.crd.yaml

```diff
@@ -23,3 +23,103 @@ spec:
   - name: Age
     type: date
     JSONPath: .metadata.creationTimestamp
+
+  validation:
+  openAPIV3Schema:
+    required: ["spec"]
+    type: object
+    properties:
+      spec:
+        type: object
+        required: ["columns", "query"]
+        properties:
+          columns:
+            type: array
+            required: ["names"]
```

Comment:
where is this property?

Reply:
I believe this line should be removed, and should be in `items`